src/app/logic/quiz.js

```diff
@@ -0,0 +1,109 @@
+const axios = require('axios');
+
+const gameInfo = {};
+
+const fetchItems = (api) => {
+  axios
+    .get(api)
+    .then((item) => item.data)
+    .then((item) => {
+      gameInfo.questionScope.push(...item.results);
+      if (item.next !== null) {
+        fetchItems(item.next);
```

Comment:
```js
Object.keys(item).includes('next') && typeof item.next === 'string' && item.next.length > process.env.API_BASE_URL
```
refactor it 😉

Reply:
### 3 steps on improved readability
1. Set as `validate`arrow fn
2. Call
3. Enjoy

Reply:
Added validate function with last two arguments.